desktop/apps/categories2/components/GeneFamilies.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
```

Comment:
Is this react convention to keep the things that are supposed to come as a list in one file? Like here GeneFamilies and GeneFamily come together?